src/key.cpp

```diff
@@ -31,7 +31,7 @@ static secp256k1_context* secp256k1_context_sign = nullptr;
  *
  * out32 must point to an output buffer of length at least 32 bytes.
  */
-static int ec_seckey_import_der(const secp256k1_context* ctx, unsigned char *out32, const unsigned char *seckey, size_t seckeylen) {
+int ec_seckey_import_der(const secp256k1_context* ctx, unsigned char *out32, const unsigned char *seckey, size_t seckeylen) {
```

Comment:
why not `CKey::Load`?

Reply:
Same here: I want to fuzz both in order to a.) maximize coverage, and b.) to avoid assuming anything about the interdependence of these two function (which may change over time).